integration/aggregate_documents_compat_test.go

```diff
@@ -1690,6 +1697,20 @@ func TestAggregateCompatProject(t *testing.T) {
 				bson.D{{"$project", bson.D{{"type", bson.D{{"$type", true}}}}}},
 			},
 		},
+		"ProjectEmpty": {
```

Comment:
This is already tested here https://github.com/FerretDB/FerretDB/blob/main/integration/aggregate_documents_test.go#L81

So no need to add compat test.